app/src/main/java/com/brainwallet/ui/screens/yourseedwords/YourSeedWordsScreen.kt

```diff
@@ -36,6 +37,7 @@ import com.brainwallet.ui.composable.BrainwalletTopAppBar
 import com.brainwallet.ui.composable.LargeButton
 import com.brainwallet.ui.composable.SeedWordItem
 
+@JvmOverloads
```

Comment:
this `@JvmOverloads` annotation not used ? since we are not calling this function from java

Reply:
Removed...this seems to have auto applied by AS.

Reply:
interesting 🤔 
thank you!

Reply:
Approval ? 😄